### PR TITLE
Dec 555 external collections

### PR DESCRIPTION
**Pull request by rfox2**
What: Added a small workflow for creating and maintaining external collections
How: Several react components were added, a new composite model was added, as well as a new controller. There are several changes to some of the pre-existing services. A URL field was added to the Exhibit model that we can use for redirects to the external sites.

### CODE REVIEW

File: app/assets/javascripts/components/forms/ButtonRaised.jsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
+/** @jsx React.DOM */
+var React = require('react');
+var mui = require("material-ui");
+var RaisedButton = mui.RaisedButton;
+var FontIcon = mui.FontIcon;
+
+var ButtonRaised = React.createClass({
```
Comment (jonhartzler) [inline app/assets/javascripts/components/forms/ButtonRaised.jsx]: I think I would like to see this class have a slightly more descriptive name.  It looks like it goes to a link.  So maybe ButtonLink or it is an action bar button link.

Comment (rfox2) [inline app/assets/javascripts/components/forms/ButtonRaised.jsx]: Changed name to "ButtonLink"

**Comment by jgondron** [comment]
:+1:

**Comment by jonhartzler** [comment]
:+1:

**Pull request merged**